config/robots.rb

```diff
@@ -3061,4 +3061,232 @@
         }
       }
     )
+
```

Comment:
these all look good to me
we just need to check with them if they want us to verify the robot barcodes